lib/get/getCache.js

```diff
@@ -18,7 +18,7 @@ function _copyCache(node, out, fromKey) {
     Object.
         keys(node).
         filter(function(k) {
-            return k[0] !== prefix && k !== "$size";
```

Comment:
I'm re-establishing this (just with charAt(0) instead of the index lookup, just to be safe), in the 0.x branch as we speak (we just hit it internally too - the refN, ref-index, etc. keys ended up in the cache).

Was just trying to collect context from @jhusain @michaelbpaulson on why we ended up going with the new impl in this PR, in 0.x, and adding unit tests, and will be ready for new PR tomorrow.